### ISSUE TITLE
Make sure that our "downloader" works behind proxies

### ISSUE THREAD

**Issue by andreasohlund** [Particular/PlatformInstaller#6]
https://github.com/Particular/PlatformInstaller/blob/develop/src/PlatformInstaller.CustomActions/CustomAction.cs#L374

**Comment by mauroservienti** [comment]
@gbiellem @SimonCropp an attendee at the Microservices Workshop last week in Rome mentioned that PI failed for him due to the authenticated proxy in his work LAN/WAN.

In this case the only workaround is a [offline installation](https://docs.particular.net/platform/installer/offline)?

**Comment by gbiellem** [comment]
@mauroservienti  yes that would be the only workaround.  
I don't suppose you have contact details or more details about the issue?

**Comment by mauroservienti** [comment]
No, unfortunately I don't. @gbiellem do you think it's possible to detect that scenario and show users a message pointing them to the offline installation instructions?

**Comment by gbiellem** [comment]
We already try too.  We have a test at startup that attempts to pull down info from S3 using no creds, then default creds, if that fails we prompt for  proxy creds.

Thinking about it more, the test item is a txt file, What that wouldn't catch is any filtering on the proxy  server to block exe downloads..  I'll reopen this issue and do some testing to ensure it works the way it's supposed to and see if we can provide better feedback

Thanks @mauroservienti

**Comment by timbussmann** [comment]
Given https://docs.particular.net/platform/installer/#what-to-expect-proxy-credentials, is this still an issue?
